core/operation/src/main/java/uk/gov/gchq/gaffer/operation/function/migration/ToUpperCase.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.operation.function.migration;
+
+import uk.gov.gchq.koryphe.function.KorypheFunction;
+
+public class ToUpperCase extends KorypheFunction<Object, String> {
```

Comment:
Perhaps this could be moved into Koryphe? Similarly with the other new functions. You could keep them in Gaffer for now until Koryphe has been updated, but perhaps just move them to src/test

Reply:
If I move all of these, for example the new Identity function, it will not be accessible in the SchemaMigration where it is currently used.  These, or at least the Identity class will need to stay accessible out of test.